### PR TITLE
config action icons

### PR DESCRIPTION
**Pull request by ahmubashshir**
- Add config file option for Action Icons.

### CODE REVIEW

File: src/NotificationCenter/Notifications/Action.hs [hunk 1]
```diff
@@ -62,7 +62,7 @@ createAction config useIcons onAction width height command description = do
   onButtonClicked button $ do
     onAction command
     return ()
-  if useIcons then do
+  if useIcons And configActionIcons config then do
```
Comment (phuhl) [inline src/NotificationCenter/Notifications/Action.hs:65]: This fails to build on my machine

Comment (phuhl) [inline src/NotificationCenter/Notifications/Action.hs:65]:
```suggestion
  if useIcons and configActionIcons config then do
```

**Review by phuhl — CHANGES_REQUESTED** [review]
Thanks for the PR, there seems to be an issue though, that prevents this from building. Also, please update the readme so, the new feature is accessible to everybody :)

File: src/NotificationCenter/Notifications/Action.hs (hunk 1 above)
Comment (ahmubashshir) [reply src/NotificationCenter/Notifications/Action.hs]: fixed :3

**Review by phuhl — CHANGES_REQUESTED** [review]
Did you try to compile this? It doesn't on my machine. I can't merge it like this. Also, still the documentation is missing.

**Comment by ahmubashshir** [comment]
> Did you try to compile this? It doesn't on my machine. I can't merge it like this. Also, still the documentation is missing.

Sorry :3 I forgot write doc.

**Comment by ahmubashshir** [comment]
Fixed it 🐱 Haskell didn't actually use `And` or `and` for logical and, 😅 Instead, it used `&&` for it 😌
And I update docs too ☺️

**Pull request merged**